app/_hub/kong-inc/forward-proxy/index.md

```diff
@@ -41,24 +41,28 @@ params:
       required: true
       default:
       value_in_examples: example.com
+      datatype: string
       description: |
         The hostname or IP address of the forward proxy to which to connect.
     - name: proxy_port
       required: true
       default:
       value_in_examples: 80
+      datatype: string
       description: |
         The TCP port of the forward proxy to which to connect.
     - name: proxy_scheme
       required: true
       default: http
```

Comment:
how do we communicate to users that only `http` is allowed? https://github.com/Kong/kong-plugin-enterprise-forward-proxy/blob/master/kong/plugins/forward-proxy/schema.lua#L21

Reply:
> how do we communicate to users that only `http` is allowed? https://github.com/Kong/kong-plugin-enterprise-forward-proxy/blob/master/kong/plugins/forward-proxy/schema.lua#L21

This is already explicitly stated: 
![image](https://user-images.githubusercontent.com/3756245/108887287-aad99b80-75cf-11eb-9394-7f05bd00696b.png)

Reply:
@Murillo going to merge since I have approval and your concern was already addressed.